### PR TITLE
feat($structure, $stylesheet, $schema): Add Description section

### PR DESCRIPTION
**Pull request by j0ntsang**

### CODE REVIEW

File: src/sections/description.liquid [hunk 1]
```diff
@@ -1,10 +1,20 @@
-<section class="description {% if section.settings.background_color_preset != 'none' and section.settings.background_color_preset != blank %}{{ section.settings.background_color_preset | append: '-section-color-preset' }}{% endif %}" data-preview-item="description">
+<section class="description large-section {% if section.settings.background_color_preset != 'none' and section.settings.background_color_preset != blank %}{{ section.settings.background_color_preset | append: '-section-color-preset' }}{% endif %}" data-preview-item="description">
```
Comment (derek-findthebrink) [inline src/sections/description.liquid:1]: not stoked on the added classname here

Comment (derek-findthebrink) [reply src/sections/description.liquid:1]: would recommend refactoring each of those section-large etc declarations in base-utils to more like
section__large (etc)

Comment (derek-findthebrink) [reply src/sections/description.liquid:1]: should be pretty quick work with find all

File: src/styles/_section_description.scss [hunk 2]
```diff
@@ -1,2 +1,41 @@
 // Description
 // ======================================
+
+.description {
+
+  .container {
+
+    &:before {
+      @include screen-lg-min {
+        left: 30%;
```
Comment (derek-findthebrink) [inline src/styles/_section_description.scss:10]: what is this one for?

**Review by derek-findthebrink — COMMENTED** [review]
looks alright, just one quick question and a suggestion on a refactor

File: src/styles/_section_description.scss (hunk 2 above)
Comment (j0ntsang) [reply src/styles/_section_description.scss:10]: This is the offset background found in elegance. This should actually be moved to the _theme.scss file at some point. Captured as part of the overall styling (for now).

File: src/sections/description.liquid (hunk 1 above)
Comment (j0ntsang) [reply src/sections/description.liquid:1]: Addressed in: https://github.com/thinkific/fundamentals/pull/4

**Pull request merged**